EnumSet initFull and initEmpty were removed from zig briefly

### Zig Version

0.16.0-dev.3132+fd2718f82

### ZLS Version

-

### Client / Code Editor / Extensions

-

### Steps to Reproduce and Observed Behavior

Try to run `zig build` on master with affected zig nightly builds.

The build will fail with two errors:

```bash
install
└─ install zls
   └─ compile exe zls ReleaseFast native 2 errors
src/Server.zig:1538:70: error: struct 'enums.EnumSet(@typeInfo(lsp_types.CodeAction.Kind).@"union".tag_type.?)' has no member named 'initEmpty'
        var set: std.EnumSet(std.meta.Tag(types.CodeAction.Kind)) = .initEmpty();
                                                                    ~^~~~~~~~~
/.../zig/zig-x86_64-linux-0.16.0-dev.3132+fd2718f82/lib/std/enums.zig:242:12: note: struct declared here
    return struct {
           ^~~~~~
referenced by:
    sendRequestSync__anon_31423: src/Server.zig:1823:68
    processMessage: src/Server.zig:1874:58
    8 reference(s) hidden; use '-freference-trace=10' to see all references
src/analysis.zig:6218:72: error: struct 'enums.EnumSet(DocumentScope.Scope.Tag)' has no member named 'initFull'
    return innermostScopeAtIndexWithTag(document_scope, source_index, .initFull()).unwrap().?;
                                                                      ~^~~~~~~~
/.../zig/zig-x86_64-linux-0.16.0-dev.3132+fd2718f82/lib/std/enums.zig:242:12: note: struct declared here
    return struct {
           ^~~~~~
error: 2 compilation errors
```

Looks like these methods were removed in https://codeberg.org/ziglang/zig/pulls/31469 and then added back a day later in https://codeberg.org/ziglang/zig/commit/be878b87c2280a3289bcd27e55843dec1ed7b568

I suppose once the next nightly is out this problem should go away, so there will be some more time to migrate away from the deprecated functions.

### Expected Behavior

The build succeeds

--- a/build.zig.zon
+++ b/build.zig.zon
@@ -10,7 +10,7 @@
     // nix flake update --commit-lock-file
     // ```
     // If you do not use Nix, a ZLS maintainer can take care of this.
-    .minimum_zig_version = "0.16.0-dev.2984+cb7d2b056",
+    .minimum_zig_version = "0.16.0-dev.3133+5ec8e45f3",
     // Must be kept in sync with the `deps.nix` for the Nix flake.
     // If you do not use Nix, a ZLS maintainer can take care of this.
     .dependencies = .{

--- a/src/Server.zig
+++ b/src/Server.zig
@@ -1535,7 +1535,7 @@ fn codeActionHandler(server: *Server, arena: std.mem.Allocator, request: types.C
     defer analyser.deinit();
 
     const only_kinds = if (request.context.only) |kinds| blk: {
-        var set: std.EnumSet(std.meta.Tag(types.CodeAction.Kind)) = .initEmpty();
+        var set: std.EnumSet(std.meta.Tag(types.CodeAction.Kind)) = .empty;
         for (kinds) |kind| {
             set.setPresent(kind, true);
         }

--- a/src/analysis.zig
+++ b/src/analysis.zig
@@ -6215,7 +6215,7 @@ pub fn innermostScopeAtIndex(
     document_scope: *const DocumentScope,
     source_index: usize,
 ) Scope.Index {
-    return innermostScopeAtIndexWithTag(document_scope, source_index, .initFull()).unwrap().?;
+    return innermostScopeAtIndexWithTag(document_scope, source_index, .full).unwrap().?;
 }
 
 pub fn innermostScopeAtIndexWithTag(

